src/main/java/com/zendesk/maxwell/bootstrap/BootstrapController.java

```diff
@@ -94,7 +94,7 @@ public synchronized void setCurrentSchemaID(long schemaID) {
 	private List<BootstrapTask> getIncompleteTasks() throws SQLException {
 		ArrayList<BootstrapTask> list = new ArrayList<>();
 		try ( Connection cx = maxwellConnectionPool.getConnection() ) {
-			PreparedStatement s = cx.prepareStatement("select * from bootstrap where is_complete = 0 and client_id = ? order by id");
+			PreparedStatement s = cx.prepareStatement("select * from bootstrap where is_complete = 0 and client_id = ? and (started_at is null or started_at <= now()) order by -started_at desc, id");
```

Comment:
sorry, why are we ordering by `-started_at desc`?   something about NULL values?

Reply:
It was meant to be, but realized that its not getting exactly. The ideia is to sort by the started_at date asc, but leaving null values at last, since they are not "scheduled". I've commited the fix checking first if the value is null or not.

Reply:
In this scenario:

<img width="529" alt="Screen Shot 2020-05-18 at 08 18 23" src="https://user-images.githubusercontent.com/52998/82207758-b7aebd80-98e0-11ea-88ca-88a3825d2cd9.png">

If we just order by started_at asc, we would get null values first as this:

<img width="536" alt="Screen Shot 2020-05-18 at 08 19 42" src="https://user-images.githubusercontent.com/52998/82207796-cb5a2400-98e0-11ea-913b-47464855584a.png">

But now, checking ifnull first in the order by we achieve the correct result

<img width="532" alt="Screen Shot 2020-05-18 at 08 19 49" src="https://user-images.githubusercontent.com/52998/82207834-da40d680-98e0-11ea-84d2-05ff6a8a337e.png">

Thanks @osheroff

Reply:
k.  reads better too.